pkg/netlink/netlink.go

```diff
@@ -372,3 +371,9 @@ func ToNetlinkFamily(family k8snet.IPFamily) int {
 
 	return netlink.FAMILY_ALL
 }
+
+func IsLinkNotFoundError(err error) bool {
+	//nolint:errorlint // netlink.LinkNotFoundError does not implement method Is(error) bool
+	_, ok := err.(netlink.LinkNotFoundError)
+	return ok || (err != nil && err.Error() == "Link not found")
```

Comment:
This doesn’t matter in current code, but to avoid surprises in future, using `errors.As` would provide support for errors wrapping `LinkNotFoundError` too:

```suggestion
	var lnferr netlink.LinkNotFoundError
	return errors.As(err, &lnferr) || (err != nil && err.Error() == "Link not found")
```